src/mutation.cpp

```diff
@@ -182,9 +182,19 @@ void Character::switch_mutations( const trait_id &switched, const trait_id &targ
     my_mutations[target].powered = start_powered;
 }
 
+bool Character::can_power_mutation( const trait_id &mut )
+{
+    if( ( mut->hunger && get_kcal_percent() < 0.5f ) || ( mut->thirst &&
+            get_thirst() >= 260 ) ||
+        ( mut->fatigue && get_fatigue() >= fatigue_levels::EXHAUSTED ) ) {
+        return false;
+    }
+    return  true;
```

Comment:
Slight style thing and/or clang-tidy might complain about the `if(x) { return false; } return true;` structure.
```suggestion
    bool hunger = mut->hunger && get_kcal_percent() < 0.5f;
    bool thirst = mut->thirst && get_thirst() >= 260;
    bool fatigue = mut->fatigue && get_fatigue() >= fatigue_levels::EXHAUSTED;

    return  hunger && fatigue && thirst;
```